spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/discovery/ServiceCombDiscoveryClientConfiguration.java

```diff
@@ -51,6 +52,12 @@ public ServiceCombDiscoveryProperties serviceCombProperties() {
     return new ServiceCombDiscoveryProperties();
   }
 
+  @Bean
+  @ConditionalOnMissingBean
+  public TagsProperties tagsProperties() {
+    return new TagsProperties();
+  }
```

Comment:
How we configure tags in servicecomb? Can do it in spring boot the same in application.yml?

Reply:
servicecomb load config by com.netflix.config.ConcurrentCompositeConfiguration .
In spring-cloud-huawei it use @ConfigurationProperties to load config. 
It's hard to say which one is better.
But is this the better to keep the same as previous code?